### PR TITLE
closes #71 | super user create admins

### LINKED ISSUE #71: As a superuser I can create Admins
Given I am authenticated and authorized as a `SUPERUSER`, When a `POST` request is made to `/auth/signup`

**Request**
```json
{
  "phone": "xxx",
  "password": "xxxxx",
  "email": "xxx"
}
```

**Response**
```json
{
  "jwt": "xxx",
  "user": {}
}
```

### PR DESCRIPTION
**Pull request by bolorundurowb**
*Description of Task*
- Allow a super user to create an admin

### CODE REVIEW

File: src/server/config/seeds/data.json [hunk 1]
```diff
@@ -19,22 +19,5 @@
         "name": "SUPER-USER"
       }
     ]
-  },
```
Comment (Saint-lawal) [inline src/server/config/seeds/data.json]: Why did you remove these from that data?

Comment (bolorundurowb) [inline src/server/config/seeds/data.json]: @Saint-lawal The reason, is because `mongoose-seed` would clear the `User` collection and seed it with that data everytime the server starts and I know thats not what we want. So I changed the implementation and placed it in the init script. Check the `Init.js` file for the way I changed it.

Comment (Saint-lawal) [inline src/server/config/seeds/data.json]: Oh okay. No p. I guess the superadmin can be used in place of the admin in terms of tests, for now.
